### ISSUE TITLE
Test workflow fails on Ubuntu 16.04 LTS

### ISSUE THREAD

**Issue by yabusygin** [yabusygin/ansible-role-docker#47]
Test workflow fails on Ubuntu 16.04 LTS. The idempotence test performed by Molecule fails on the task "install dependencies of Ansible modules for Docker".

**Comment by yabusygin** [comment]
When trying to install these packages for the second time manually (`pip3 install 'docker>=1.10.0' 'PyYAML>=3.11' 'docker-compose>=1.7.0'`), we can see the following output:

```
<...>
Collecting pyrsistent>=0.14.0 (from jsonschema<4,>=2.5.1->docker-compose>=1.7.0)
<...>
Installing collected packages: pyrsistent
  Found existing installation: pyrsistent 0.0.0
    Uninstalling pyrsistent-0.0.0:
      Successfully uninstalled pyrsistent-0.0.0
Successfully installed pyrsistent-0.0.0
```

**Comment by yabusygin** [comment]
As a workaround add the following task bеfore the role:

```yaml
- pip:
  name:
    - pyrsistent==0.14.0
```

**Comment by yabusygin** [comment]
Closed by #48
